idx/memory/memory.go

```diff
@@ -220,25 +246,20 @@ func (m *MemoryIdx) indexTags(def *schema.MetricDefinition) {
 		tags.addTagId(tagName, tagValue, id)
 	}
 	tags.addTagId("name", def.Name, id)
+
+	m.DefByTagSet.add(def)
 }
 
 // deindexTags takes a given metric definition and removes all references
 // to it from the tag index. It assumes a lock is already held.
-func (m *MemoryIdx) deindexTags(def *schema.MetricDefinition) {
-	tags, ok := m.tags[def.OrgId]
-	if !ok {
-		corruptIndex.Inc()
-		log.Error(3, "memory-idx: corrupt index. ID %q can't be removed. no tag index for org %d", def.Id, def.OrgId)
-		return
-	}
-
+func (m *MemoryIdx) deindexTags(tags TagIndex, def *schema.MetricDefinition) bool {
```

Comment:
please document what the bool means

Reply:
done: https://github.com/grafana/metrictank/pull/806/commits/8021face0dafa51e4084b172cda1810579f2401a